packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2058,24 +2058,30 @@ export class LitNodeClientNodeJs extends LitCore {
       return this.getClaimKeyExecutionShares(url, nodeRequestParams, requestId);
     });
 
-    const responseData = await this.handleNodePromises(nodePromises);
+    const responseData = await this.handleNodePromises(
+      nodePromises,
+      this.connectedNodes.size // require from all connected nodes
```

Comment:
we will fix the contracts to not need all nodes for derivation, and then we can change this.